meta-resin-common/recipes-core/initrdscripts/files/resindataexpander

```diff
@@ -6,9 +6,22 @@ datadev=$(lsblk $datapart -n -o PKNAME)
 
 resindataexpander_enabled() {
     # On flasher avoid expanding partition
-    if [ -h /dev/disk/by-label/flash-boot ]; then
-        echo "[INFO] Flasher detected. Avoiding expand partition mechanism."
-        return 1
+	if [ -h /dev/disk/by-label/flash-rootA ]; then
+		flash_root=$(realink -f "/dev/disk/by-label/flash-rootA")
```

Comment:
`realink` or `readlink`?

Reply:
Good catch. Haven't tested it yet though. So keep it pending for a while.